packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -213,9 +213,9 @@ export const Components = {
     items: 'Search items',
   },
   DashboardLinks: {
-    container: 'Dashboard link container',
-    dropDown: 'Dashboard link dropdown',
-    link: 'Dashboard link',
+    container: 'data-testid Dashboard link container',
```

Comment:
🤔 is it intentional to have the data-testid?

Reply:
Yes. It's for the e2e tests. If it starts with `data-testid` it will use that attribute instead of `aria-label`.